Gateway/Command/PayByMailCommand.php

```diff
@@ -136,18 +136,8 @@ protected function getFormFields($payment, $paymentAmount = false)
         $realOrderId = $order->getRealOrderId();
         $storeId = $order->getStore()->getId();
 
-        // check if paybymail has it's own skin
-        $skinCode = trim($this->_adyenHelper->getAdyenPayByMailConfigData('skin_code'));
-        if ($skinCode == "") {
-            // use HPP skin and HMAC
-            $skinCode = $this->_adyenHelper->getAdyenHppConfigData('skin_code');
```

Comment:
@ataberkylmz I think the adyen_hpp skin_code was removed from the config and that's why you're removing this, right?

Reply:
@acampos1916 Indeed! adyen_hpp/skin_code did not exist anymore so having the check there does not make sense. Same goes for the adyen_hpp/delivery_days.

Reply:
Great. So if this is not being used anywhere else maybe you can check if this issue is solved or if something extra is needed https://github.com/Adyen/adyen-magento2/issues/783

Reply:
It is resolved to some extend by removing the non-existent configs, however if you leave it empty, you'll still get that error. We can use `validate-no-empty` validator but this would force merchants to enter a value there so I'd really recommend it.